ledger/archival_test.go

```diff
@@ -64,6 +64,10 @@ func (wl *wrappedLedger) Block(rnd basics.Round) (bookkeeping.Block, error) {
 }
 
 func (wl *wrappedLedger) BlockHdr(rnd basics.Round) (bookkeeping.BlockHeader, error) {
+	return wl.blockHdr(rnd)
+}
+
+func (wl *wrappedLedger) blockHdr(rnd basics.Round) (bookkeeping.BlockHeader, error) {
```

Comment:
this function `blockHdr` not needed

Reply:
Still needed. will remove in the next PR (Part II)